packages/owl-bot/src/app.ts

```diff
@@ -15,15 +15,15 @@
 // eslint-disable-next-line node/no-extraneous-import
 import {Probot} from 'probot';
 import {GCFBootstrapper} from 'gcf-utils';
-import {OwlBot} from './owl-bot';
+import {owlbot} from './owl-bot';
 const bootstrap = new GCFBootstrapper();
 // Unlike other probot apps, owl-bot-post-processor requires the ability
 // to generate its own auth token for Cloud Build, we use the helper in
 // GCFBootstrapper to load this from Secret Manager:
 module.exports.owl_bot = bootstrap.gcf(
   async (app: Probot) => {
     const config = await bootstrap.getProbotConfig(false);
-    OwlBot(config.privateKey, app);
+    owlbot.OwlBot(config.privateKey, app);
```

Comment:
Pulled exported methods from OwlBot service onto `owlbot` object, so that it's easier to mock.